debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerStreamingChangeEventSource.java

```diff
@@ -360,56 +356,17 @@ private void migrateTable(final Queue<SqlServerChangeTable> schemaChangeCheckpoi
                                 dataConnection.getTableSchemaFromTable(currentTable),
                                 SchemaChangeEventType.CREATE));
             }
+            //
+            // TODO: This needs to be re-worked per https://github.com/debezium/debezium/pull/748#issuecomment-492526200
```

Comment:
@jpechane, is this todo tracked in Jira?

Reply:
@jpechane, ok, so I'm confused. This refers to the very comment that was the reason why @grzegorz8's PR wasn't merged in the first place. I just don't know: is this an actual issue now or not?